runtime/compiler/rpc/J9Client.hpp

```diff
@@ -23,8 +23,6 @@
 #ifndef J9_CLIENT_H
 #define J9_CLIENT_H
 
-#include <google/protobuf/io/zero_copy_stream_impl.h>
-#include "compile/CompilationTypes.hpp"
 #include "rpc/StreamTypes.hpp"
 #include "rpc/ProtobufTypeConvert.hpp"
 #include "j9.h"
```

Comment:
Is it possible to remove this `j9.h` or include something directly from the OMR layer? The goal is to contribute this communications layer to OMR, so we want to eliminate anything that is J9 related.